Shoko.Server/API/v3/Models/Shoko/Series.cs

```diff
@@ -451,14 +495,19 @@ public class AniDB
         /// <summary>
         /// The TvDB Data model for series
         /// </summary>
-        public class TvDB
+        public class TvDB : ISeriesMetadata
         {
             /// <summary>
             /// TvDB ID
             /// </summary>
             [Required]
             public int ID { get; set; }
 
+            [JsonConverter(typeof(StringEnumConverter))]
+            public SeriesType Type { get { return SeasonNumber == 0 ? SeriesType.TVSpecial : SeriesType.TVSeries; } }
+
+            public bool Restricted { get { return false; } }
```

Comment:
Probably best to add docs to the added params

Reply:
TvDB doesn't allow Restricted content. That's why it's not listed. Also why I argue specific models per provider

Reply:
Removed the interfaces, and reverted most of the cosmetic changes.